File: lib/models/kaji.dart
/// 家事のデータモデル
class Kaji {
  final int id;
  final String title;
  final String content;
  final int points;
  DateTime? recordedDate; // 家事実施日時（任意）

  Kaji({
    required this.id,
    required this.title,
    required this.content,
    required this.points,
    this.recordedDate,
  });
}

File: backend/models/kaji.go
package models

import "time"

type Kaji struct {
	ID        uint      `gorm:"primaryKey" json:"id"`
	Title     string    `json:"title"`
	Content   string    `json:"content"`
	Points    int       `json:"points"`
	CreatedAt time.Time `json:"created_at"`
	UpdatedAt time.Time `json:"updated_at"`
}